src/buildstream_plugins/elements/autotools.py

```diff
@@ -51,6 +51,14 @@
          conf-global: |
            --disable-gtk-doc --disable-static
 
+By default build-dir is not used. To improve probability build does not
+tamper with sources, you can set build-dir like
+
+.. code:: yaml
+
+   variables:
+     build-dir: bst_build-dir
```

Comment:
Nit: this is a strange suggestion with mixed `-` and `_`

perhaps `_build` ?

Reply:
Ah, yes. True.

Reply:
I guess to be consistent with cmake and meson we could suggest _builddir